src/tests/electron/common/view-controllers/automated-checks-view-controller.ts

```diff
@@ -64,4 +65,14 @@ export class AutomatedChecksViewController extends ViewController {
 
         await this.waitForSelector(toggleInStateSelector);
     }
+
+    public async clickLeftNavItem(key: LeftNavItemKey): Promise<void> {
+        const selector = this.getSelectorForLeftNavItemLink(key);
+        await this.waitForSelector(selector);
+        await this.client.click(selector);
```

Comment:
Would be preferable to use `this.click` here instead of `this.client.click`; that way it'll get wrapped in `screenshotOnError` for you